cronus/worker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM jupyter/scipy-notebook
```

Comment:
Is this a docker image from a repository?

Reply:
Yes, docker images without a hostname refer to docker hub.  So this image is: https://hub.docker.com/r/jupyter/scipy-notebook/.  If you want to refer to a different container repository, like Google Container Repository, you need to prefix with the hostname (and google project), like this: gcr.io/broad-ctsa/my-image.